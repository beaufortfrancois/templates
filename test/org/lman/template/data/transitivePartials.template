{{#context}}
Outer content says "hello": {{hello}}.
{{+content:partials.p1}}
Inner content which uses variable from context "hello": {{hello}}.
{{/partials.p1}}
{{/context}}
