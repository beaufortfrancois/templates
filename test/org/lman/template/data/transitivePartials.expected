Outer content says "hello": world.
Rendering p1...

Inner content which uses variable from context "hello": world.
